### ISSUE TITLE
report for survival models

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/report#150]
> Similar to report package natural language summaries, I have also tried to generate summary sentences from survival models. I wanted to share it here and ask for comments, since you labelled this issue as `what's your opinion`.

> If you find them useful, I will try to format them as new report method.

> Code here:
https://github.com/sbalci/ClinicoPathJamoviModule/blob/master/R/survival.b.R

<img width="642" alt="pairwise-survival-summary" src="https://user-images.githubusercontent.com/7208830/81602576-8a8c7780-93d5-11ea-9ff2-72419e490d85.png">

<img width="659" alt="survival-summary" src="https://user-images.githubusercontent.com/7208830/81602587-8eb89500-93d5-11ea-831f-12928be29d0b.png">

_Originally posted by @sbalci in https://github.com/easystats/report/issues/60#issuecomment-626908324_

**Comment by m-macaskill** [comment]
Yes please, it would be great to add support for both `survfit` and `coxph` objects from the `survival` package.
